packages/debug/src/browser/debug-frontend-application-contribution.ts

```diff
@@ -241,6 +243,10 @@ export namespace DebugCommands {
         id: 'editor.debug.action.showDebugHover',
         label: 'Debug: Show Hover'
     });
+    export const JUMP_TO_CURSOR = Command.toDefaultLocalizedCommand({
+        id: 'editor.debug.action.jumpToCursor',
+        label: 'Debug: Jump to Cursor'
+    });
```

Comment:
Suggestion: We usually split the label from the category. I'm not sure why this isn't done for the `SHOW_HOVER` command.
```suggestion
    export const JUMP_TO_CURSOR = Command.toDefaultLocalizedCommand({
        id: 'editor.debug.action.jumpToCursor',
        category: DEBUG_CATEGORY,
        label: 'Jump to Cursor'
    });
```

Reply:
done